src/routes/boot.ts

```diff
@@ -328,6 +331,48 @@ const getUser = (con: DataSource, userId: string): Promise<User> =>
     ],
   });
 
+const getMarketingCta = async (con: DataSource, userId: string) => {
+  let marketingCta: MarketingCta | null = null;
+  const redisKey = generateStorageKey(
+    StorageTopic.Boot,
+    'marketing_cta',
+    userId,
+  );
+  const rawRedisValue = await getRedisObject(redisKey);
+
+  if (rawRedisValue === RedisMagicValues.SLEEPING) {
+    // TODO: remove log or replace with proper logging
+    console.log('sleeping, not fetching from db');
+    return null;
+  } else {
+    marketingCta = JSON.parse(rawRedisValue);
+  }
+
+  if (!marketingCta) {
+    // TODO: remove log or replace with proper logging
+    console.log('fetching from db');
```

Comment:
This one you can remove, kind of normal behavior

Reply:
Will remove, but keeping it for now as debugging